<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lTerm">
     <property name="text">
      <string>&amp;Terminal:</string>
     </property>
     <property name="buddy">
      <cstring>leTerm</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leTerm"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="cbMinOnOpen">
     <property name="toolTip">
      <string>Minimize on open connection</string>
     </property>
     <property name="text">
      <string>Minimize on &amp;open</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cbUseTray">
     <property name="toolTip">
      <string>Use system tray icon</string>
     </property>
     <property name="text">
      <string>Use &amp;tray</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="cbStartHidden">
     <property name="toolTip">
      <string>Start programm in hidden mode</string>
     </property>
     <property name="text">
      <string>Start &amp;hidden</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="cbMinToTray">
     <property name="toolTip">
      <string>Hide program to tray on minimize button</string>
     </property>
     <property name="text">
      <string>&amp;Minimize to tray</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
